mobile/voice_journal_app/lib/RecordingPage.dart

```diff
@@ -0,0 +1,250 @@
+late DateTime presently; //what day/time is it presently? went with the shortest name I could think of
+int secondsCounter = 0;
+late Timer _timer;
+String message = "Press the record button to start";
+Icon rcrdIcon = const Icon(Icons.mic_off_outlined);  
+bool recording = false; //Establish a bool to keep track of button state
+bool transcribed = false; //Transcribed bool **just a placeholder for now**
+late Directory appDirectory; //late meanning initializing later in code, but deffining it now
+  bool counting = false; //Counting bool to track if the counter has been started or not. Unsure if I can stop it so this bool is used to prevent double trigger (counting on 2s)
+// ------Done Initializing Variables----
+void displaySaved(){ //Create a little pop up letting the user know their reccording has been saved
+  Fluttertoast.showToast(
+    msg: 'saved',
+    toastLength: Toast.LENGTH_SHORT,
+    gravity: ToastGravity.BOTTOM,
+    timeInSecForIosWeb: 1,
+    backgroundColor: const Color.fromARGB(0, 76, 175, 79),
+    textColor: Colors.white,
+    fontSize: 16.0,
+  );
+}
+startRecording() async{
+  rcrdIcon = const Icon(Icons.pause);
+  recording = true;
+  final hasPermission = await controller.checkPermission(); //get permission to use mic
+  //appDirectory = await getApplicationDocumentsDirectory();  //get the apps directory useful later
+  appDirectory = Directory('/storage/emulated/0/Download'); //Set directory for file to go to
+  String exactDirectory = appDirectory.path;
+  presently = DateTime.now(); //Set presently string to date and time
+  String formattedDateTime = DateFormat('yyy-MM-dd-HH-mm-ss').format(presently); //Format date and time to work as a valid file name
+  path = '$exactDirectory' +'/$formattedDateTime.m4a';//Set the path to where it should go + the current date and time
+  if(hasPermission){ //If we got phone permissions
+    controller.record(path: path); //Record
+  }
+  message = 'Recording'; //change message to show recording
+}
+stopRecording(){
+  if(counting){
+    counting = false;//Don't count anymore
+    _timer.cancel();
+  }
+    recording = false; //No longer recording
+    controller.stop(); //Stop the recording controller
+    secondsCounter = 0; //reset counter to 0
+    rcrdIcon = const Icon(Icons.mic_off_outlined);  
+    message = 'Recording Stopped';
+}
+pauseRecording(){
+  recording = false; //not recording
+  controller.pause(); //pause the recording
+  rcrdIcon = const Icon(Icons.play_arrow);
+  message = 'Recording paused';
+  displaySaved();
+}
+
+void main() {
+  runApp(const MainApp());
+}
+class MainApp extends StatelessWidget {
+  const MainApp({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'recording page',
+      theme: ThemeData( //Themes
+        colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepOrange),
+        useMaterial3: true,
+      ),
+      home: const RecordingPage(title: "recordingPage"),
+    );
+  }
+}
```

Comment:
There's no need for this starting point in the app since we already have `main.dart` and `RecordingPage` will be called when navigated to.